### ISSUE TITLE
React 19 Support

### ISSUE THREAD

**Issue by chioio** [mutativejs/zustand-mutative#5]
Could you please consider upgrading to support React 19? Thank you!

**Comment by chioio** [comment]
```json
{
  "peerDependencies": {
    "immer": ">=8.0.0",
    "react": "^16.8.0 || ^17.0.1 || ^18.0.0 || ^19.0.0"
  },
}

**Comment by unadlib** [comment]
hi @chioio , I will update zustand-mutative later to support React 19.

**Comment by unadlib** [comment]
I have released zustand-mutative v1.2.0. Feel free to use it.
